jobdispatcher/src/test/java/com/firebase/jobdispatcher/ConstraintTest.java

```diff
@@ -16,18 +16,21 @@
 
 package com.firebase.jobdispatcher;
 
-import static org.junit.Assert.assertEquals;
-
 import android.text.TextUtils;
-import java.util.Arrays;
-import java.util.List;
+
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
-@RunWith(RobolectricGradleTestRunner.class)
-@Config(constants = BuildConfig.class, manifest = Config.NONE, sdk = 21)
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
```

Comment:
nit: preferred import style is static imports (sorted alphabetically), a blank line, then everything else (sorted alphabetically)